components/dashboard/src/components/PillLabel.tsx

```diff
@@ -4,12 +4,14 @@
  * See License.AGPL.txt in the project root for license information.
  */
 
-export type PillType = "info" | "warn" | "success";
+export type PillType = "info" | "warn" | "success" | "warnStrong" | "subtle";
 
 const PillClsMap: Record<PillType, string> = {
     info: "bg-blue-50 text-blue-500 dark:bg-blue-500 dark:text-blue-100",
```

Comment:
**thought**: Going with the same subtle (neutral) label could be nice with the upcoming changes in https://github.com/gitpod-io/gitpod/pull/15389, see relevant designs in https://github.com/gitpod-io/gitpod/issues/15287#issuecomment-1357671514. Cc @svenefftinge 

Alternatively, using the info variant could be also a good way forward. But in any case, going with the neutral and improving the label designs (colors) (https://github.com/gitpod-io/gitpod/issues/15218) in the future sounds good. 

| Using neutral label variant |  Using info label variant |
|-|-|
| <img width="712" alt="Frame 269" src="https://user-images.githubusercontent.com/120486/208451915-0f72c384-b4f3-4346-8882-1a776f205a9a.png"> | <img width="712" alt="Frame 269-1" src="https://user-images.githubusercontent.com/120486/208451920-cc26ceec-5ace-4b75-b72b-ae04dfc7d02d.png"> |